src/IQFeed.CSharpApiClient/Lookup/Historical/Messages/DailyWeeklyMonthlyMessage.cs

```diff
@@ -77,7 +77,7 @@ public override int GetHashCode()
             unchecked
             {
                 var hash = 17;
-                hash = hash * 29 + RequestId.GetHashCode();
+                hash = hash * 29 + RequestId != null ? RequestId.GetHashCode() : 0;
```

Comment:
could you explain why you think requestid worth adding it?

Reply:
the RequestId was already there, I just fixed a NullException here :) I didn't want to remove it
maybe we can remove it after all

Reply:
what do you think about this?